docs/guides/protect-page-login/php.mdx

```diff
@@ -0,0 +1,135 @@
+---
+id: php
+title: Protect a Page with Login in PHP
+sidebar_label: PHP
+---
+
+```mdx-code-block
+import Teaser from './_common/teaser.mdx'
+import mp4 from './go/screencast.mp4'
+import webm from './go/screencast.webm'
+
+<Teaser
+  framework={<a href="https://www.php.net/">PHP</a>}
+  mp4={mp4}
+  webm={webm}
+/>
+```
+
```

Comment:
Please add a small introduction paragraph like we do here https://www.ory.sh/docs/guides/protect-page-login/expressjs